src/components/TradeWidget/Price.tsx

```diff
@@ -84,7 +84,7 @@ export const PriceInputBox = styled.div`
     }
   }
 
-  label > div {
+  label > div:not(.radio-container) {
```

Comment:
Maybe rather than target everything but (`:not(*)`), better target a specific class.
Otherwise this might bite us back at another time

Or not, doesn't matter much

Reply:
yeah i also considered this but this the only use place where it differs, i can change it if others think it important